.mailmap

```diff
@@ -536,8 +536,9 @@ Sandon Van Ness <svanness@redhat.com> <sandon@redhat.com>
 Sangdi Xu <xu.sangdi@h3c.com>
 Sangdi Xu <xu.sangdi@h3c.com> <xu.sangdi@h3c>
 Scott A. Brandt <scott@cs.ucsc.edu> <sbrandt@29311d96-e01e-0410-9327-a35deaab8ce9>
-Sebastian Wagner <swagner@suse.com> <sebastian.wagner@suse.com>
-Sebastian Wagner <swagner@suse.com> <sebastian@spawnhost.de>
+Sebastian Wagner <sewagner@redhat.com> <sebastian.wagner@suse.com>
+Sebastian Wagner <sewagner@redhat.com> <swagnerr@suse.com>
+Sebastian Wagner <sewagner@redhat.com> <sebastian@spawnhost.de>
```

Comment:
i am not quite sure about this. before this change both as the work of `sebastian.wagner@suse.com` and `sebastian@spawnhost.de` was sponsored by SUSE. as both mail addressed are associated with `Sebastian Wagner <swagner@suse.com>`  which is in turn connected to SUSE by the entry of
```
SUSE <contact@suse.com> Sebastian Wagner <swagner@suse.com>
```
in `.organizationmap`.

but after this change, even the work of `<sebastian.wagner@suse.com>` is categorized to `Sebastian Wagner <sewagner@redhat.com>`, which is in turn mapped to `Red Hat <contact@redhat.com>`. 

so i guess, it's wrong. and i cannot find a commit authored by `swagnerr@suse.com` in ceph/ceph. so at least, i think we should keep the line of

```
Sebastian Wagner <swagner@suse.com> <sebastian.wagner@suse.com>
```

unchanged. and drop
```
Sebastian Wagner <sewagner@redhat.com> <swagnerr@suse.com>
```

see also https://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide#Mail-notification-to-the-author-about-an-update